### PR TITLE
Support (old) IKEA Tradfri Wireless Dimmer ICTC-G-1

### PR DESCRIPTION
**Pull request by TheJulianJES**
Similar to the IKEA SYMFONISK "dimmer", but doesn't support clicks.
- adds doubling of the battery percentage: fixes https://github.com/zigpy/zha-device-handlers/issues/557, https://github.com/home-assistant/core/issues/42672
- also adds basic device automation triggers for moving the dial left and right

The support is for an IKEA wireless dimmer now with the updated ZB3 firmware. I didn't include the previous quirk for the ZLL firmware.
Based upon https://github.com/zigpy/zha-device-handlers/issues/557 and tested by @MattWestb

### CODE REVIEW

File: zhaquirks/ikea/dimmer.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LEFT,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    RIGHT,
+)
+
+ROTATED = "device_rotated"
```
Comment (TheJulianJES) [inline zhaquirks/ikea/dimmer.py:34]: This is required for the ``device_automation_triggers`` and is also present in the ``symfonisk.py``.
Perhaps this should be moved into the ``__init.py__`` in the IKEA "folder" and then be imported in both quirks?

Comment (Adminiuga) [reply zhaquirks/ikea/dimmer.py:34]: either move it into `__init__.py` or you could do `from .sympfonisk import ROTATED`

**Review by Adminiuga — APPROVED** [review]
LGTM

**Comment by TheJulianJES** [comment]
I moved ``device_rotated`` to the IKEA ``__init__.py`` file now as that seems *cleaner*.
It does look like that Black reformatted some parts of the file though. One second

**Comment by MattWestb** [comment]
Was updating the 3 changed files after doing the last chopping round this year and the quirk is loading OK and battery is reported as expected !!
HA is firing ZHA events from cluster 0x0008 commands so its looks doing wath is expected :-))

Thanks for great coding and one great end and happy new year with greetings from [Miss Sophie and James](https://en.wikipedia.org/wiki/Dinner_for_One) ;-))

**Pull request merged**